src/main/resources/arguments.json

```
@@ -214,6 +214,11 @@
       "_type": "required"
     }
   },
+  "seek": {
```

```suggestion
  "seek*": {
```
To avoid duplicates
And fix this: https://gyazo.com/3b02a4e76e8f0a85a03e18c1514bf5e6

---

Not sure what's causing that issue actually, from what I can see everything's correct in lang and arguments. Help appreciated.

---

Just figured it out, woop